src/configuration/configuration.cpp

```diff
@@ -258,6 +262,8 @@ void Configuration::write() const {
 
   conf.setValue("Static room description parser type", (int)m_roomDescriptionsParserType);
   conf.setValue("Minimum static lines in room description", m_minimumStaticLines);
+
+  conf.setValue("MUME charset is UTF-8", m_utf8Charset);
```

Comment:
Setting encoding and color settings with MPI would be awesome, also because it would kill two prompts for new players with compatible clients.

This may be more complicated though, as both these settings have to match what the client behind MMapper expects. So the client would have to negociate it and MMapper would have to snoop that exchange and adapt its settings...